config/build/rollup.config.js

```diff
@@ -11,7 +11,7 @@ import ancesdir from "ancesdir";
 import autoExternal from "rollup-plugin-auto-external";
 import copy from "rollup-plugin-copy";
 import filesize from "rollup-plugin-filesize";
-import less from "rollup-plugin-less";
+import styles from "rollup-plugin-styles";
```

Comment:
`styles` handles `.css` as well as `.less` files. The `less` plugin _only_ handles `.less` files.